### ISSUE TITLE
Rename tsv folder to 'segments' and json folder to 'matches'

### ISSUE THREAD

**Issue by sebastian-nehrdich** [BuddhaNexus/buddhanexus#217]
Its no good practice to call these folders based on the filetype they contain; more explicit naming is what we need! Also requires codechanges in the dataloader to make sure the new paths are read correctly.

**Comment by angirov** [comment]
Do you mean this:
![image](https://github.com/BuddhaNexus/buddhanexus/assets/19487247/9245efbe-f77b-40c6-90ab-7b3d1c9d5a22)

**Comment by sebastian-nehrdich** [comment]
Yes and (more importantly) in the buddhanexus backend folder as well to
make the structure more clear. That reminds me that we should rename “data”
which holds tib-files.json etc to “metadata” to make it more clear as well…

**Comment by sebastian-nehrdich** [comment]
agree!

**Comment by sebastian-nehrdich** [comment]
feel happy to add this in one of the upcoming PRs, no rush for now
